src/random.cr

```diff
@@ -56,6 +56,14 @@ module Random
   # The integers must be uniformly distributed between 0 and the maximal value for the chosen type.
   abstract def next_u : UInt
 
+  # Generates a slice filled with *n* random bytes.
+  def random_bytes(n : Int) : Bytes
```

Comment:
Buffering would solve many problems that were kind of worked around. But... that just seems not... secure. So I avoided it on purpose.